charts/zservice/values.yaml

```diff
@@ -1,22 +1,60 @@
-postgres:
-  addr: zsrv-postgresql:5432
-  username: "zqd"
-  database: "zqd"
-  # passwordSecretName must have a value called postgres-password
-  passwordSecretName: "postgres"
-recruiter:
-  replicaCount: 1
-root:
-  datauri: "file:///spaces"
-  recruiterAddr: zsrv-recruiter:9867
-worker:
-  recruiterAddr: zsrv-recruiter:9867
-
 global:
   AWSRegion: ""
   image:
     repository: "localhost:5000/"
     pullPolicy: Always
     tag: "zqd"
   imagePullSecrets: []
+  postgres:
+    addr: zsrv-postgresql:5432
+    username: "zqd"
+    database: "zqd"
+    # passwordSecretName must have a value called postgres-password
+    passwordSecretName: "postgres"
   useCredSecret: false
+
+postgresql:
+  # These values only apply to postgres instance running in the cluster (not Aurora)
+  # Possible values: standalone, replication.
+  architecture: standalone  
+  # Username, password, database for zqd cluster use.
+  postgresqlUsername: "zqd"
+  postgresqlDatabase: "zqd"
+  existingSecret: "postgres"  
+  image:
+    tag: "11"  
+  # Enable persistence using Persistence Volume Claims.
+  persistence:
+    enabled: true  
+    accessModes:
+      - ReadWriteOnce 
+    size: 8Gi
+
+recruiter:
+  replicaCount: 1
+
+redis:
+  usePassword: true
+  existingSecret: redis
+  image:
+    tag: "6.0"
+  cluster:
+    enabled: false
+  # set maxmemory + maxmemory-policy in order to use redis as an lru cache. See
+  # https://redis.io/topics/lru-cache
+  maxmemory: 1gb
+  maxmemoryPolicy: volatile-lfu
+  configmap: |
+    maxmemory {{ .Values.maxmemory }}
+    maxmemory-policy {{ .Values.maxmemoryPolicy }}
+
+root:
+  datauri: "file:///spaces"
+  recruiterAddr: zsrv-recruiter:9867
+
+worker:
+  recruiterAddr: zsrv-recruiter:9867
+
+tags:
+  deploy-postgres: true
```

Comment:
I'm trying to see how setting this to false stops postgres from being installed?

Reply:
The tags feature in Helm is tricky to follow, the tag here is referenced in Chart.yaml:

  - name: postgresql
    repository: https://charts.bitnami.com/bitnami
    version: 10.x.x
    tags: 
      - deploy-postgres

Reply:
Should the `deploy-redis` section be under `tags` too, then?

Reply:
Yes, tags.deploy-redis is defined in values.yaml and defaults to true.